src/Orleans/Configuration/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Orleans.Runtime.Configuration
+{
+    public static class ConfigurationExtensions
```

Comment:
Since we are going to revamp config in the near future, I wanted to avoid adding more surface that we'll break later. I understand the extension methods are still in the public surface, but still it feels like breaking it less if we re-do that type. I might be wrong.